test/unittests/test_xml_upload_stash.py

```diff
@@ -0,0 +1,89 @@
+# pylint: disable=missing-class-docstring,missing-function-docstring,protected-access
+
+from unittest import TestCase
+
+import pytest
+
+import dsp_tools.utils.xml_upload_stash as upld
```

Comment:
I'm not a fan of importing modules under a different name, because it confuses me when I read "upld" further down in the code. I normally import functions in this way:

```
from path.to.module import function1, function2, ...
```